client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -305,6 +311,28 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			} );
 		}
 
+		if ( EditorActions.GetNavSidebarLabels === action ) {
+			const { translate } = this.props;
+
+			ports[ 0 ].postMessage( {
+				allPostsLabels: {
+					page: translate( 'View all pages' ),
+					post: translate( 'View all posts' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **77** times:
`translate( 'View full post' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/nav-sidebar-view-all-pages)